src/mson.js

```diff
@@ -125,7 +125,7 @@ function handleDescription(description, element, parent, spaces, marker) {
 
   // Now, depending on the content type, we will recursively handle child
   // elements within objects and arrays.
-  if (element.content && element.content.length) {
+  if (element.content && Array.isArray(element.content) && element.content.length) {
```

Comment:
Strings have `length` property too, this was falsely allowing to continue with a string and use `forEach` on it which raises exceptions since a string doesn't have `forEach`.